src/Pages/License/Generate/FeatureDetails.jsx

```diff
@@ -1,9 +1,147 @@
+  flexDirection: "column",
+  justifyContent: "center",
+}));
+
+function createData(feature, currentSetting, originalSetting, fieldType) {
+  return { feature, currentSetting, originalSetting, fieldType };
 }
 
-export default FeatureDetails
+const rows = [
+  createData("Central Issuance Smart Card", "On", "Off", "Select"),
+  createData("Ice cream sandwich", "235", "9", "Input"),
+  createData("Eclair", "123", "16", "Input"),
+  createData("Cupcake", "7543", "3.7", "Input"),
+  createData("Gingerbread", "2", "16", "Input"),
+];
+
+const licenseKeys = ["Plus", "Professional", "Enterprise"];
+
+const FeatureDetails = (props) => {
+  const [alignment, setAlignment] = useState("Professional");
+  const changeProductKey = (newAlignment) => {
+    setAlignment(newAlignment);
+  };
+
+  return (
+    <Grid item container direction="column" {...props}>
+      <CustomCard>
+        <CustomCardContent>
+          <Grid item mb={1}>
+            <Typography variant="h6">Product Edition</Typography>
+          </Grid>
+          <Grid item mb={2}>
+            <Typography variant="caption">
+              Pick the edition to batch edit the product features
+            </Typography>
+          </Grid>
+
+          <ToggleButtonGroup exclusive aria-label="text alignment">
+            <Stack spacing={3} direction="row">
+              {licenseKeys.map((key) => (
+                <ToggleButton
+                  value={key}
+                  aria-label="centered"
+                  key={key}
+                  selected={key === alignment}
+                  style={{ color: "#87189D", borderColor: "#87189D" }}
+                  onClick={(e) => {
+                    changeProductKey(e.target.value);
+                  }}
+                >
+                  {key}
+                </ToggleButton>
+              ))}
+            </Stack>
+          </ToggleButtonGroup>
+        </CustomCardContent>
+      </CustomCard>
+      <CustomCard>
+        <CustomCardContent>
+          <TableContainer component={Paper}>
+            <Table
+              sx={{ minWidth: 600 }}
+              aria-label="simple table"
+              size="small"
+            >
+              <TableHead>
+                <TableRow>
+                  <TableCell>Feature</TableCell>
+                  <TableCell>Current Setting</TableCell>
+                  <TableCell>Original Setting</TableCell>
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {rows.map((row) => (
+                  <TableRow
+                    key={row.feature}
+                    sx={{ "&:last-child td, &:last-child th": { border: 0 } }}
+                  >
+                    <TableCell component="th" scope="row">
+                      {row.feature}
+                    </TableCell>
+                    <TableCell align="left">
+                      {row.fieldType === "Select" && (
+                        <NativeSelect id="demo-customized-select-native">
```

Comment:
Add ids to all UI elements (input fields, button, checkbox, radio etc)

Reply:
done